docs/core/build/distribution-packaging.md

```diff
@@ -17,49 +17,109 @@ ms.workload:
 
```

Comment:
Should we add a sentence mentioning the target audience for this document For example: Users attempting to build from source or making changes that would impact building from source (such as the layout/etc).

Reply:
Adding target audience is always desirable. Can you expand on what you mean by:

> making changes that would impact building from source (such as the layout/etc).

Reply:
I would imagine that the people who care about this document are:
* New users onboarding to the team
* Users attempting to build from source
* Users wanting to make changes to the CLI that could impact the resulting layout or packages produced (basically anyone making changes that would impact the things this document is spec'ing out)